nemo_text_processing/text_normalization/es/verbalizers/electronic.py

```diff
@@ -58,21 +58,23 @@ def add_space_after_char():
 
         verbalize_characters = pynini.cdrewrite(graph_symbols | graph_digit, "", "", NEMO_SIGMA)
 
-        user_name = pynutil.delete("username: \"") + add_space_after_char() + pynutil.delete("\"")
+        user_name = pynutil.delete('username: "') + add_space_after_char() + pynutil.delete('"')
         user_name @= verbalize_characters
 
         convert_defaults = pynutil.add_weight(NEMO_NOT_QUOTE, weight=0.0001) | domain_common | server_common
         domain = convert_defaults + pynini.closure(insert_space + convert_defaults)
         domain @= verbalize_characters
 
-        domain = pynutil.delete("domain: \"") + domain + pynutil.delete("\"")
+        domain = pynutil.delete('domain: "') + domain + pynutil.delete('"')
         protocol = (
-            pynutil.delete("protocol: \"")
+            pynutil.delete('protocol: "')
             + add_space_after_char() @ pynini.cdrewrite(graph_symbols, "", "", NEMO_SIGMA)
-            + pynutil.delete("\"")
+            + pynutil.delete('"')
         )
-        self.graph = (pynini.closure(protocol + pynini.accep(" "), 0, 1) + domain) | (
-            user_name + pynini.accep(" ") + pynutil.insert("arroba ") + domain
+
+        self.graph = (pynini.closure(protocol + NEMO_SPACE, 0, 1) + domain) | (
+            user_name + NEMO_SPACE + pynutil.insert("arroba ") + domain | (pynutil.insert("arroba ") + user_name)
```

Comment:
Just alias "arroba " with a variable name to avoid future spelling issues

Reply:
Done.